near-sdk-sim/src/units.rs

```diff
@@ -18,3 +18,30 @@ pub fn to_yocto(value: &str) -> u128 {
         part1
     }
 }
+
+pub fn to_gas(tera_gas: &str) -> u64 {
```

Comment:
An alternative would be 

```rust
pub fn tera_gas(tera_gas: f64) -> u64
```

Long term (definitelly out of scope for this PR), for SDK specifically, I feel like we should have a dedicated `Gas` type and write `Gas::from_tera_f64(1.1)`, to follow std's Duration type: https://doc.rust-lang.org/stable/std/time/struct.Duration.html#method.from_secs_f32.